Pipeline/WordPress/Stages/Category/VicCategoryGeneratorStages.php

```diff
@@ -3,7 +3,7 @@
 namespace Victoire\DevTools\VacuumBundle\Pipeline\WordPress\Stages\Category;
 
 use Doctrine\ORM\EntityManager;
-use Victoire\Bundle\BlogBundle\Entity\Category;
+use Victoire\Bundle\BlogBundle\Entity\BlogCategory;
```

Comment:
`Category` changed to `BlogCategory` in the `2.2.29` release of victoire, it should be reflected in composer.json, requiring `^2.2.29`.